WKWebView.ios.js

```diff
@@ -254,6 +254,7 @@ var WKWebView = React.createClass({
     });
 
     var source = this.props.source || {};
+    source = Object.assign(source, { sendCookies: this.props.sendCookies });
```

Comment:
Aren't props read-only? E.g., by supplying a read-only prop as the target to `Object.assign`, I believe a runtime exception would be thrown.